packages/patternfly-4/react-core/src/components/Progress/Progress.docs.js

```diff
@@ -0,0 +1,34 @@
+import { Progress } from '@patternfly/react-core';
+import Simple from './examples/SimpleProgress';
+import LargeProgress from './examples/LargeProgress';
+import ProgressFailure from './examples/ProgressFailure';
+import ProgressFailureWithoutMeasure from './examples/ProgressFailureWithoutMeasure';
+import ProgressInside from './examples/ProgressInside';
+import ProgressInsideSuccess from './examples/ProgressInsideSuccess';
+import ProgressOutside from './examples/ProgressOutside';
+import ProgressOutsideFailure from './examples/ProgressOutsideFailure';
+import ProgressSuccess from './examples/ProgressSuccess';
+import ProgressWithoutMeasure from './examples/ProgressWithoutMeasure';
+import SmallProgress from './examples/SmallProgress';
+import ProgressWithDynamicDescription from './examples/ProgressWithDynamicDescription';
+
+export default {
+  title: 'Progress',
+  components: {
+    Progress
+  },
+  examples: [
+    Simple,
+    SmallProgress,
+    LargeProgress,
+    ProgressOutside,
+    ProgressInside,
+    ProgressSuccess,
+    ProgressFailure,
+    ProgressInsideSuccess,
+    ProgressOutsideFailure,
```

Comment:
Looking at the live output, in this example, the error icon does not appear to be vertically aligned with the text

Reply:
Yeah all icons from `react-icons` has this nasty bug, when I was looking trough the styling it looks like positioning is misplaced, but I don't know the right way to position them correctly.